src/ssl.c

```diff
@@ -15401,6 +15498,17 @@ void wolfSSL_X509_STORE_CTX_set_error(WOLFSSL_X509_STORE_CTX* ctx, int err)
     (void)err;
 }
 
+#ifdef OPENSSL_EXTRA
+void wolfSSL_X509_STORE_CTX_set_time(WOLFSSL_X509_STORE_CTX* ctx,
```

Comment:
Yes, this time is used to validate CERT time and CRL time in OpenSSL.
(x509_check_cert_time() and check_crl_time() in OpenSSL's crypto/x509/x509_vfy.c)
So I added struct WOLFSSL_X509_VERIFY_PARAM in struct WOLFSSL_X509_STORE_CTX structure.(wolfssl/ssl.h)
I just define only 2 members (check_time and flags) along with OpenSSL in struct WOLFSSL_X509_VERIFY_PARAM.
Might need more member in future.
If you validate certificates with setting time value during SSL\TLS handshake,need some update,I guess.
I am not familiar with CertManager but I guess it's same.
I investigate OpenSSL code how to check certificate time.
If you set time by X509_STORE_CTX_set_time,use setting value for validation.
If you don't set time,it validate with current time.